src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -20,6 +24,10 @@
 
   private static final String SMART_BANNER_SELECTOR = ".smartbanner.android";
   private static final String FLITE_MASK_SELECTOR = ".flite-mask";
+  private static final String MERCURY_WIKI_TITLE_SELECTOR = ".wiki-title a";
+  private static final String WIKI_LINK_SELECTOR = "a[href^='/wiki/']";
+  private static final String MERCURY_LOADING_OVERLAY = "loading-overlay";
```

Comment:
Good point. And... there is an error too. I wonder how it worked with this error :japanese_ogre: I think it should be `.loading-overlay`. Let me fix it... :smile_cat: